contracts/dnsregistrar/DNSRegistrar.sol

```diff
@@ -99,7 +105,7 @@ contract DNSRegistrar {
 
     function supportsInterface(bytes4 interfaceID) external pure returns (bool) {
         return interfaceID == INTERFACE_META_ID ||
-               interfaceID == DNSSEC_CLAIM_ID;
```

Comment:
Do you still need the following DNSSEC_CLAIM_ID? It doesn't seem to be used anywhere

```
  bytes4 constant private DNSSEC_CLAIM_ID = bytes4(
        keccak256("claim(bytes,bytes)") ^
        keccak256("proveAndClaim(bytes,bytes,bytes)") ^
        keccak256("oracle()")
    );
```